src/server/index.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+const { createClient } = require('./sproutVideo');
+const { ELEMENT_STATE } = require('../shared');
+const omit = require('lodash/omit');
+
+async function afterSave(asset, { config: { tce } }) {
+  const { videoId, playable, fileName, error, status } = asset.data;
+  if (!fileName || playable) return asset;
+  const { sproutVideoApiKey: apiKey } = tce;
+  const client = createClient({ apiKey });
+  const isEmptyOrError = error || !videoId || status !== ELEMENT_STATE.UPLOADED;
+  if (!isEmptyOrError) startPollingPlayableStatus(asset, client);
```

Comment:
Would it work if we'd simplify this condition like this:
```suggestion
  if (status === ELEMENT_STATE.UPLOADED) startPollingPlayableStatus(asset, client);
```
?

Reply:
Due to fact that we don't have an `ERROR` state, I think there is still a need to check the `error` prop to see if there is an error before pooling for playable status. 
We can omit videoId check. What do you think?